ocifs/tests/test_spec.py

```diff
@@ -1507,3 +1508,69 @@ def test_sync(fs):
         assert len(fs.ls(remote_dir)) == len(os.listdir(tmpdirname))
 
     fs.rm(remote_dir, recursive=True)
+
+
+@pytest.mark.parametrize(
+    "isImplicit, content_type", [(True, "image/jpeg"), (False, "image/jpeg")]
+)
+def test_content_type_image(fs, isImplicit, content_type):
+    from PIL import Image
+
+    import io
+
+    e_path = "image.jpeg"
+    e = os.path.join(full_test_bucket_name, e_path)
+
+    image = Image.new("RGBA", size=(50, 50), color=(256, 0, 0))
+    image_file = io.BytesIO(image.tobytes())
+
+    if isImplicit:
```

Comment:
If/else inside the test case is confusing. Can you break this into two test cases.